src/test/java/uk/gov/digital/ho/hocs/queue/common/integration/MigrationQueueListenerIntegrationTest.java

```diff
@@ -49,4 +49,28 @@ public void consumeMessageFromMigrationQueueWithOptionalFieldsNull() {
         String validMessage = TestFileReader.getResourceFileAsString("validMigrationWithOptionalFieldsNull.json");
         amazonSQSAsync.sendMessage(migrationQueueUrl, validMessage);
     }
+
+    @Test
+    public void consumeMessageFromMigrationQueueWithCOMPCaseType() {
```

Comment:
These tests aren't actually testing anything I don't think and have been removed from my other PR altogether till an appropriate implementation is thought of.

Reply:
Yes that's a good point.  We use these primarily for local testing, consuming a message from case-creator then making sure it goes to casework fine - creating all the case, correspondents, case-attachments as necessary in the database and verify in the frontend, so it's more a test-utility rather than a test, but they are helpful.  

I realise that we shouldn't have code that isn't strictly used, and this is more for in-dev testing, but could we leave these in until the majority of the right hand side tickets are done?